docs/installation/slack/cloud-slack.mdx

```diff
@@ -74,6 +74,22 @@ After this point, all of your commands will be executed on the default instance.
 
 ![Cloud Slack All Clusters](assets/cloud_slack_command_all_clusters.png "Cloud Slack all clusters")
 
+### Setting Public Alias for Private Channels
+
+In order to safeguard your confidentiality while enabling seamless integration with Botkube plugins, it's essential to establish a public alias for your private Slack channels. This alias will be accessible solely to Botkube Cloud administrators.
+
+#### During the Botkube Invitation to Private Channels
+
+Upon extending an invitation to Botkube for a private channel, a prompt will appear, guiding you through the process of creating a public alias. This step ensures smooth communication and integration.
```

Comment:
change this: When you invite Botkube to a private channel, a prompt will appear to guide you through the process of creating a public alias.

Reply:
resolved in https://github.com/kubeshop/botkube-docs/pull/321